pkg/node/node.go

```diff
@@ -113,6 +112,7 @@ type Bee struct {
 	chainSyncerCloser        io.Closer
 	shutdownInProgress       bool
 	shutdownMutex            sync.Mutex
+	syncing                  chan struct{}
```

Comment:
So I am not sure if we should name this `syncing`

I got confused while going through the PR about what this is meant for. Maybe `postageSyncingFailure` would be better? or along those lines?

Reply:
it's basically going to be open as long as there's a live listening goroutine, it's not related to failure handling, just blocking the main thread from exiting.
maybe can be named to something like `syncingInProgress`

Reply:
I leave it up to you to rename this or not. Its just that we will in the code we will exit when this channel is closed. So it looks more like a signal to stop which is why I would prefer it to be some error type string.
`syncingStopped` maybe. But its up to you.